LEGO1/mxpalette.cpp

```diff
@@ -70,9 +70,9 @@ MxResult MxPalette::SetSkyColor(LPPALETTEENTRY p_entries)
 void MxPalette::GetDefaultPalette(LPPALETTEENTRY p_entries)
 {
   HDC hdc = GetDC((HWND) NULL);
-  int rasterCaps = GetDeviceCaps(hdc, RASTERCAPS);
+  MxS32 rasterCaps = GetDeviceCaps(hdc, RASTERCAPS);
```

Comment:
`GetDeviceCaps` is a Windows function. It returns `int`, hence the local variable storing the return value from it should also be plain `int`.

The Mx primitive types are for Mx interfaces (return values and parameters) and member data.